app/lib/tariff_synchronizer.rb

```diff
@@ -288,11 +289,19 @@ def sync_variables_set?
   end
 
   def oplog_based_models
-    Sequel::Model.subclasses.select do |model|
+    sequel_models.select do |model|
       model.plugins.include?(Sequel::Plugins::Oplog)
     end
   end
 
+  def sequel_models
+    # Sequel::Model subclasses need to load into the ruby AST before they are visible
+    # This only affects running this code in development mode which does not eager load in the normal course of events
+    Rails.autoloaders.main.eager_load if Rails.env.development?
```

Comment:
This is using `Rails.env.development?` to imply that eager loading is disabled, its better to be explicit and use that configuration value directly, eg `unless Rails.configuration.eager_load`